meta_review/load_from_db.py

```diff
@@ -0,0 +1,192 @@
+import logging
+
+from meta_review.models import Participant, Comment, Reaction
+
+
+def load_all(participants, comments, reactions):
+    """
+    Load all meta-review related data to memory.
+    """
+    load_participants(participants)
+    load_comments(participants, comments)
+    load_reactions(participants, comments, reactions)
+
+
+def load_participants(participants):
+    """
+    Load participants into memory.
+
+    a) create Participant objects if not exist in database
+       fetch history data if exist in database and also in memory
+    b) fetch history data if exist in database but not in memory yet
+    """
+    logger = logging.getLogger(__name__)
+    logger.info('get or create participants')
+    created_cnt = 0
+    existing_cnt = 0
+    load_cnt = 0
+    # There are lots of participants and we'd better use bulk_create
+    # to accelerate deploy process
+    old_participants = Participant.objects.all()
+    old_participants_set = set()
+    for old_participant in old_participants:
+        old_participants_set.add(old_participant.login)
+
+    new_participants = []
+    for participant in participants.values():
+        # if it is an old participant, we skip it
+        if participant['login'] in old_participants_set:
+            logger.debug('participant %s exists' % participant['login'])
+            existing_cnt += 1
+        else:
+            logger.debug('participant %s is new' % participant['login'])
+            new_participants.append(
+                Participant(login=participant['login']))
+            created_cnt += 1
+
+    # use bulk create to speed up create process
+    Participant.objects.bulk_create(new_participants)
+
+    # load all participants again (old + new)
+    all_participants = Participant.objects.all()
+
+    for p in all_participants:
+        if participants.get(p.login):
+            # participants recently active
+            participant = participants[p.login]
+            p.name = participant['name']
+            # save into memory
+            participants[p.login] = p
```

Comment:
`participants[p.login] = p` is common to both branches, so could be after the `if`